.vib/kibana/cypress/cypress/integration/kibana_spec.js

```diff
@@ -21,113 +21,44 @@ it('allows uploading a file', () => {
     );
   });
   cy.get('[data-test-subj="dataVisualizerFileImportButton"]').click();
-  cy.contains('span', 'Index created').scrollIntoView();
-  cy.contains('span', 'Data uploaded').scrollIntoView();
-  cy.contains('span', 'Data view created');
   cy.contains(
     '[data-test-subj="dataVisualizerFileImportSuccessCallout"]',
     'Import complete'
   );
 });
 
-it('allows creating a dashboard', () => {
-  cy.visit('app/dashboards/create/');
-  closeThePopups();
-  cy.contains('button', 'Create').forceClick();
-  cy.get('[data-test-subj="dashboardSaveMenuItem"]')
-    .should('have.text', 'Save')
-    .forceClick();
-  cy.fixture('dashboards').then((dashboard) => {
-    cy.get('[data-test-subj="savedObjectTitle"]').type(
-      `${dashboard.newDashboard.title}.${random}`
-    );
-    cy.get('[data-test-subj="dashboardDescription"]').type(
-      `${dashboard.newDashboard.description}.${random}`
-    );
-  });
-  cy.contains(
-    '[data-test-subj="confirmSaveSavedObjectButton"]',
-    'Save'
-  ).forceClick(); //forcing because there is a popup we can't control
-  cy.visit('/app/dashboards/list');
-  cy.fixture('dashboards').then((dashboard) => {
-    cy.contains(`${dashboard.newDashboard.title}.${random}`);
-  });
-});
-
-it('allows creating a visualisation', () => {
+it('allows creating a dashboard and visualisation', () => {
   cy.visit('app/visualize/');
   closeThePopups();
   cy.contains('button', 'Create').click();
```

Comment:
Use the `data-test-subj` attribute whenever possible.

Reply:
Modified every affected check.